### PR TITLE
raise on missing checkpoint path

### PR DESCRIPTION
**Pull request by GeorgWa**
Raise error on missing checkpoint path

### CODE REVIEW

File: alphadia/constants/default.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ library_prediction:
   # maximum charge state for predicted fragments
   max_fragment_charge: 2
   instrument: Lumos
-  checkpoint_folder_path: None
+  checkpoint_folder_path: null
```
Comment (mschwoer) [inline alphadia/constants/default.yaml:44]: could you add a short comment here: what behaviour does 'null' mean, why would I set this?

also: really `null`?

File: alphadia/libtransform.py [hunk 2]
```diff
@@ -314,6 +314,11 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
 
         model_mgr = ModelManager(device=device)
         if self.checkpoint_folder_path is not None:
+            if not os.path.exists(self.checkpoint_folder_path):
```
Comment (mschwoer) [inline alphadia/libtransform.py:317]: is `checkpoint` really a good name? I confused it with the `.progress` concept ..

File: alphadia/constants/default.yaml (hunk 1 above)
Comment (GeorgWa) [reply alphadia/constants/default.yaml:44]: It's stupid, I agree. `null` in a yaml translates into `None` in python. `None` gets parsed as `'None'` 😭

File: alphadia/libtransform.py (hunk 2 above)
Comment (GeorgWa) [reply alphadia/libtransform.py:317]: This is how it's called within peptdeep at the moment. We could name it `peptdeep_model_path`, what do you think?

Comment (mschwoer) [reply alphadia/libtransform.py:317]: sounds good

File: alphadia/constants/default.yaml (hunk 1 above)
Comment (mschwoer) [reply alphadia/constants/default.yaml:44]: again what learned ..
could we change this to 
`checkpoint_folder_path: # empty: use default ~/peptdeep`

Comment (GeorgWa) [reply alphadia/constants/default.yaml:44]: Yep, thats the current behavior with `null`.
We couuuld, but in yaml `key: ` gets parsed as `key: null`.  😄 so maybe the verbose version is better?

**Review by mschwoer — APPROVED** [review]

**Pull request merged**